cli/daemon/dash/dashapp/src/components/trace/SpanDetail.tsx

```diff
@@ -271,7 +271,9 @@ const GoroutineDetail: FunctionComponent<{g: gdata, req: Request, trace: Trace,
     setBarOver(true)
     setHoverObj(obj)
     const spanEl = (ev.target as HTMLElement)
-    el.style.marginTop = `calc(${spanEl.offsetTop}px - 40px)`;
+    const offset = spanEl.getBoundingClientRect();
```

Comment:
`getBoundingClientRect` gives you offset relative to your client viewport, `offsetTop` did never change in this case.